src/core/title.js

```diff
@@ -69,6 +75,19 @@ function setDocumentTitle(conf, h1Elem) {
     documentTitle = `Preview of PR #${conf.prNumber}: ${documentTitle}`;
   }
 
+  if (conf.hasOwnProperty("level")) {
+    if (validLevel(conf.level)) {
```

Comment:
we should probably normalize the level here... so: 
```suggestion
    const level = parseInt(conf.level);
    if (validLevel(level)) {
```

Reply:
`parseInt(num)` has unexpected things such as 
```parseInt("1t")
1```

thus, I'd rather avoid that and stick to `Number.isInteger` and `>=`

Reply:
I think that's fine. We don't need to overcheck.

Reply:
could you help me understand what `parseInt` does that the checks for validLevel (`Number.isInteger` and `>=`) don't? I'm not sure why it is needed in addition to those two checks. thanks!

Reply:
> could you help me understand what parseInt does that the checks for validLevel (Number.isInteger and >=) don't?

So, first thing:

 Number.isInteger("1") === false 

but we are generally pretty liberal with allowing simple values to be canonicalized. The other thing that we get is an simple assurance of a number or NaN:

```
const level = parseInt(conf.level, 10);
```

So it's mostly just for string handling, really.